step.yml

```diff
@@ -0,0 +1,33 @@
+title: Validate JSON schema
+summary: Run JSON schema validation.
+
+website: https://github.com/bitrise-steplib/steps-validate-json-schema
+source_code_url: https://github.com/bitrise-steplib/steps-validate-json-schema
+support_url: https://github.com/bitrise-steplib/steps-validate-json-schema/issues
+
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/steps-validate-json-schema
+
+inputs:
+- schema_url:
+  opts:
+    title: JSON schema URL
+    summary: URL of the JSON schema to use.
```

Comment:
Does this accept local file URLs too? If yes I would add a note

Reply:
Added in: https://github.com/bitrise-steplib/steps-validate-json-schema/pull/1/commits/e9e553ae7119b1f5a4c1b82ac700dc9ee05689a7